dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -788,6 +792,9 @@ BlockInputStreamPtr Segment::getInputStream(
     UInt64 max_version,
     size_t expected_block_size)
 {
+    Stopwatch watch;
+    SCOPE_EXIT({ dm_context.scan_context->create_inputstream_time_ns += watch.elapsed(); });
```

Comment:
IMHO, it is not quite necessary

Reply:
The time here contains building delta-index and building bitmap. Most time should be covered by building bitmap, removed.